### ISSUE TITLE
Add `assets list` command

### ISSUE THREAD

**Issue by aminamos** [muxinc/cli#35]

**Comment by mmcc** [comment]
What's the use-case for this in a CLI environment?

**Comment by aminamos** [comment]
I'm not sure if this fits the direction of the CLI, but there's a few helper scripts related to this that I didn't want to forget. Some variation of "perform [create/list/update/delete] on [x] or all assets"

**Comment by eropple** [comment]
Being able to pipe-xargs to do asset stuff seems totally cromulent to me.

**Comment by mmcc** [comment]
Yeah, I can see it in theory, my concern was mostly based on what you said @aminamos, which is that since [filtering the asset list call](https://docs.mux.com/api-reference/video#operation/list-assets) is pretty limited, the use cases really become "stuff to do against *all* assets". Given that, I'd been leaning more in the camp of using the CLI for more developer experience related activities/utilities and trying things out vs a load bearing production tool to delete an entire environment.

That being said, I'm totally open to it, that's just where my head was in drawing the line around what endpoints we initially exposed here. If we're going to go this route we also _definitely_ need to make sure we've got a `--json` flag to make it easier for folks to pipe responses and work with them in this way. Right now everything is pretty human oriented.

**Comment by eropple** [comment]
IMO `--json` is in this case less useful than being able to rip with unix tools, but yeah, we need better filtering in general (which we've mostly ignored because none of these resource calls are particularly useful in that environment right now as-is.

Having a way to just brrrrap through a problem without having to open up a text editor to write a one-off script, when bash is right there, is probably a big usability win in general.
